src/ext/handlersToResp.c

```diff
@@ -204,6 +204,9 @@ static RdbRes resolveSupportRestore(RdbParser *p, RdbxToResp *ctx) {
     /* Enforce RESTORE for modules. librdb cannot really parse high-level module object */
     RDB_handleByLevel(p, RDB_DATA_TYPE_MODULE, RDB_LEVEL_RAW);
 
+    /* Avoid RESTORE for functions. Redis doesn't have API to RESTORE functions */
```

Comment:
that's not true.
redis does have FUNCTION DUMP and FUNCTION RESTORE.
it has the same format as the RDB file, i.e. dumping dumping all functions at once.
so in theory instead of sending multiple FUNCTION LOAD commands, you can buffer that data and send s single FUNCTION RESTORE command.
not sure what's the benefit of that (maybe if there are a ton of functions (libraries), which is unlikely IMHO).

Maybe the benefit of such, is that you can use the FLUSH argument, and there's no need for an additional FUNCTION FLUSH before starting.

which brings me to realize that where we use FLUSHALL, we should also use FUNCTION FLUSH, and where we use DEL or REPLACE, we should do the same for functions. please look into it.

Reply:
Missed that one. Thanks. I will expose configuration flag to decide whether to REPLACE or not on FUNCTION LOAD.